manifest.konnector

```diff
@@ -58,5 +45,18 @@
       "type": "io.cozy.accounts",
       "verbs": ["GET"]
     }
+  },
+  "developer": {
+    "name": "Cozy Cloud",
+    "url": "https://cozy.io"
+  },
+  "langs": ["fr"],
+  "locales": {
+    "fr": {
+      "short_description": "Récupère toutes vos factures Bouygues Box"
```

Comment:
Permissions are not translated ?

Reply:
These were not translated before. Now this is done